README.md

```diff
@@ -3,3 +3,31 @@
 Extract data from all types of Australian driving licenses and Australian and international passports and validate the id (i.e. check it hasn't been altered or tampered with).
 
 ## Set Up
+
+Flask - venv setup  
+start by downloading the venv as well as getting the same dependencies:
```

Comment:
Should this say "downloading" or "creating"?

Reply:
O i used the word downloading because i had the idea of getting the venv module into our directory from the python libraries, but creating is a better word. Ayt will have it changed